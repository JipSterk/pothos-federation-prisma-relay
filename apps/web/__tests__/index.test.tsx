import { render } from "@testing-library/react";
import React from "react";
import Home from "../src/app/page";

describe("Home", () => {
  it.skip("should render a homepage", () => {
    const { asFragment } = render(<Home />);
    expect(asFragment()).toMatchSnapshot();
  });
});
