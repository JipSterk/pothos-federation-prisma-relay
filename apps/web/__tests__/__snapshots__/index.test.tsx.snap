// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render a homepage 1`] = `
<DocumentFragment>
  <main />
</DocumentFragment>
`;
